docs/multinode-metrics.md

```diff
@@ -1 +1,34 @@
+## Metrics to enhance the grafana for multi-node etcd
```

Comment:
Can you please refine this and describe what the purpose of this document is? It's not quite accurate to use "Garfana" here because it's only one way to visualize the captured metrics. This is rather a document of metrics that we consider important to observe the runtime behavior of multi-node etcd clusters. How it's scraped, stored and visualized is up to the operator.